backend/photos/models.py

```diff
@@ -42,8 +42,8 @@ class Photo(UUIDModel, VersionedModel):
     metering_mode                       = models.CharField(max_length=32, null=True)
     drive_mode                          = models.CharField(max_length=32, null=True)
     shooting_mode                       = models.CharField(max_length=32, null=True)
-    camera                              = models.ForeignKey(Camera, related_name='photos', null=True)
-    lens                                = models.ForeignKey(Lens, related_name='photos', null=True)
+    camera                              = models.ForeignKey(Camera, related_name='photos', null=True, on_delete=models.CASCADE)
+    lens                                = models.ForeignKey(Lens, related_name='photos', null=True, on_delete=models.CASCADE)
```

Comment:
I'm happy with the model changes regarding on_delete and glad there wasn't too much to change with Django upgrade.